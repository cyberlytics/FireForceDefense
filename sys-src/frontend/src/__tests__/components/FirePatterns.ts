import { mount } from '@vue/test-utils';
import FirePatterns from '@components/firePatterns.vue';

describe('FirePatterns.vue', () => {
    test('fire patterns match snapshot', () => {
        const wrapper = mount(FirePatterns);
        expect(wrapper.element).toMatchSnapshot();
    });
});
