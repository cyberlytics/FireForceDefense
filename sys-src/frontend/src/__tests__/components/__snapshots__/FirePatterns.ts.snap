// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirePatterns.vue fire patterns match snapshot 1`] = `
<defs>
  <pattern
    height="1"
    id="fire-1"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/fire_intensity1.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="fire-2"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/fire_intensity2.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="fire-3"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/fire_intensity3.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="fire-4"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/fire_intensity4.svg"
      y="0"
    />
  </pattern>
</defs>
`;
